examples/ACE/fit-neural-ace.jl

```diff
@@ -36,7 +35,13 @@ args = ["experiment_path",      "a-Hfo2-300K-NVT-6000-NeuralACE/",
         "wL",                   "1.0",
         "csp",                  "1.0",
         "w_e",                  "0.01",
-        "w_f",                  "1.0"]
+        "w_f",                  "1.0",
+        "device",              "gpu"]
```

Comment:
The default should be cpu, because today we do not have a complete gpu implementation.